connector/src/test/java/com/google/cloud/spark/bigquery/SparkBigQueryConfigTest.java

```diff
@@ -41,7 +42,14 @@ public void testDefaults() {
     Configuration hadoopConfiguration = new Configuration();
     DataSourceOptions options = new DataSourceOptions(defaultOptions);
     SparkBigQueryConfig config =
-        SparkBigQueryConfig.from(options, ImmutableMap.of(), hadoopConfiguration, 10);
+        SparkBigQueryConfig.from(
+            options,
+            ImmutableMap.of(),
+            hadoopConfiguration,
+            10,
+            new SQLConf(),
+            "2.4.0",
```

Comment:
might be good to have this as a constant since it has been used at multiple places